tests/test_regression.py

```diff
@@ -1,100 +1,333 @@
 #! /usr/bin/env python
 
+from nose.tools import assert_equal, assert_true
```

Comment:
import groupings - third party library libraries should be below standard libraries, with a single whitespace separator.